integration/deploy_test.go

```diff
@@ -55,6 +55,28 @@ var _ = Describe("Deploy", func() {
 	})
 
 	Context("when incorrectly setup", func() {
+		It("should report correct error if ops for is incorrect", func() {
+			tearDown, err := env.CreateConfigMap(env.Namespace, env.DefaultBOSHManifest("manifest"))
+			Expect(err).NotTo(HaveOccurred())
+			defer tearDown()
+
+			tearDown, err = env.CreateConfigMap(env.Namespace, env.InterpolateOpsConfigMap("bosh-ops"))
+			Expect(err).NotTo(HaveOccurred())
+			defer tearDown()
+
+			tearDown, err = env.CreateSecret(env.Namespace, env.InterpolateOpsIncorrectSecret("bosh-ops-secret"))
+			Expect(err).NotTo(HaveOccurred())
+			defer tearDown()
+
+			boshDeployment, tearDown, err := env.CreateBOSHDeployment(env.Namespace, env.InterpolateBOSHDeployment("test", "manifest", "bosh-ops", "bosh-ops-secret"))
+			Expect(err).NotTo(HaveOccurred())
+			defer tearDown()
+
+			success, result := env.GetBOSHDeploymentEventMessage(env.Namespace, boshDeployment.ObjectMeta.Name, string(boshDeployment.ObjectMeta.UID))
+			Expect(success).To(BeTrue())
+			Expect(result).To(ContainSubstring("Expected to find exactly one matching array item for path '/instance-groups/name=api' but found 0"))
```

Comment:
Does this test one of the events from `reconciler.go`? Which one?

Reply:
It tested this event: https://github.com/cloudfoundry-incubator/cf-operator/blob/f5abafcd7742f3eeecef3b4305279f1140ce18b6/pkg/kube/controllers/boshdeployment/reconciler.go#L79

I'll update other events